sed/core/logging.py

```diff
@@ -19,26 +21,55 @@
 
 def setup_logging(
```

Comment:
I generally think we need to set the default logging to INFO and allow for an easy way to change that to debug. Maybe something like this way, the user can change to their preference. 
```
from sed.logging import VERBOSITY
VERBOSITY = 10 # to set to debug
``` 

Maybe doesn't make sense to log inside log files, unless specified by user.

Reply:
I did not like the idea to expose the loglevels directly to the user, thus the verbosity flag. There should still be a setter for it that updates the loglevel globally.

Reply:
I added a setter for the verbosity level now.